index.html

```diff
@@ -12,7 +12,14 @@
 </head>
 <body>
     <div class="container">
-
+        <footer id="footer">
+            <h2>Game Instructions</h2>
+            <div>The aim of the game is to find all matching <span>pairs</span>,</div>
+            <div>Click on two <span>squares</span> to see the devs who are there.</div>
+            <div>Each turn, try to <span>remember</span> their places,</div>
+            <div>When you have found a <span>matching</span> pair you will still see their faces.</div>
```

Comment:
Why a div for each line? I think it's better to wrap the whole thing in one <p> tag and use line breaks.